src/screens/WhotoFollow/index.js

```diff
@@ -141,21 +133,6 @@ const WhotoFollow = () => {
     setFollowed(copyFollowed);
   };
 
-  const onRefresh = React.useCallback(() => {
-    setRefreshing(true);
-    get({url: '/who-to-follow/list'})
-      .then((res) => {
-        setRefreshing(false);
-        if (res.status === 200) {
-          setUsers(res.data.body);
-        }
-      })
-      .catch((err) => {
-        crashlytics().recordError(new Error(err));
-        setRefreshing(false);
-      });
-  }, []);
-
   const register = async () => {
     setFetchRegister(true);
     Analytics.logEvent('onb_select_follows_btn_add', {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onRefresh` function has been removed. This function was responsible for refreshing the data when the user pulls down on the list. If this functionality is required, it should be reimplemented.
```diff
- const onRefresh = React.useCallback(() => {
-   setRefreshing(true);
-   get({url: '/who-to-follow/list'})
-     .then((res) => {
-       setRefreshing(false);
-       if (res.status === 200) {
-         setUsers(res.data.body);
-       }
-     })
-     .catch((err) => {
-       crashlytics().recordError(new Error(err));
-       setRefreshing(false);
-     });
- }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->